docs/x509/ocsp.rst

```diff
@@ -340,7 +340,11 @@ Creating Responses
             :class:`~cryptography.hazmat.primitives.asymmetric.ed448.Ed448PrivateKey`
             and an instance of a
             :class:`~cryptography.hazmat.primitives.hashes.HashAlgorithm`
-            otherwise.
+            otherwise. This should not be the same algorithm used to add
+            a certificate to the response, as
+            :class:`~cryptography.hazmat.primitives.hashes.SHA1` can't be used
+            for signing, despite its compatibility requirement for certain 
+            applications.
```

Comment:
I'd suggest rephrasing to state that this does not need to be the same algorithm used in `add_response` (i.e., link to it) and separately that SHA1 is disallowed here even though it's allowed here.

We don't want people to think that if they pass `SHA256` to `add_response` that they shouldn't use it here.

Reply:
I think I rephrased it well enough?